R/read_csv.R

```diff
@@ -321,6 +331,7 @@ read_cmdstan_csv <- function(files,
     }
     list(
       metadata = metadata,
+      time = list(total = NA_integer_, chains = time),
```

Comment:
this keeps the structure consistent with the output of fit$time() but `total` is `NA` because we can't infer it from the CSV files